Procfile

```diff
@@ -1,2 +1,2 @@
 web: gunicorn wsgi:application -b 0.0.0.0:$PORT -w 8
-worker: python manage.py celeryd -E -B --beat --concurrency=1
\ No newline at end of file
+worker: celery worker --app helios --events --beat --concurrency 1 --logfile celeryw.log --pidfile celeryw.pid
```

Comment:
Yes. From Kombu 4 the django backend for jobs isn't supported anymore, and I didn't find a replacement. So users should setup one of the supported brokers (RabbitMQ is an example).
I've added now the configurability via system env of CELERY_BROKER_URL

Reply:
Hi guys! So, if I understood it right, we now have a final answer for this thread:
https://groups.google.com/forum/#!topic/helios-voting/GVBKOHFTC5E
i.e, we all reached the conclusion that is not possible to use django backend anymore if upgrade is intended? @benadida is nice to hear from you again! There are some good PR waiting for your attention :D

Reply:
I didn't know of that google group, nice!
Yes, the drawback highlighted by @benadida here and in that thread (the additional piece of software required) is necessary. It would be really nice to find an alternative, but no easy one is around.
Probably, something like this could be implemented, but I didn't check how to integrate that behaviour with celery+django: https://layerci.com/blog/postgres-is-the-answer/

Reply:
I didn't search deep enough, previously.
Someone extracted again the django-kombu package outside of kombu itself on github: [barseghyanartur/django-kombu](https://github.com/barseghyanartur/django-kombu). Probably some work with that could be done, also integrating celery/kombu#536 since we support only postgres.
Since I think it could be nice, in this time quarantined I could make some effort, but to gain the most result for us, I'll make it compatible just for py3 and django3 (but I'll try to keep it compatible with previous versions of django-kombu integration, so users of helios could migrate easily)

Reply:
@McCio that makes perfect sense to me, and it would be lovely. I also really like the article you pointed to – I'd much rather not add another component to deploying Helios, especially given that the number of jobs that need to be run by the background queue is typically quite small, just compute intensive.

Reply:
> Hi guys! So, if I understood it right, we now have a final answer for this thread:
> https://groups.google.com/forum/#!topic/helios-voting/GVBKOHFTC5E
> i.e, we all reached the conclusion that is not possible to use django backend anymore if upgrade is intended? @benadida is nice to hear from you again! There are some good PR waiting for your attention :D

Hi @shirlei I'm sorry I've been too absent here. Please do ping me on the PRs you think are most important to merge – some of them are likely outdated and I'm sorry about that!